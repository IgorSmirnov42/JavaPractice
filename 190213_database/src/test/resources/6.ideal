Write command name (8 for help): Write owner name: Write phone number: Write command name (8 for help): Write owner name: Write phone number: Write command name (8 for help): Write owner name: Write phone number: Write new phone number: Write command name (8 for help): Write owner name: Write phone number: Write new phone number: Write command name (8 for help): a b
b t
Write command name (8 for help): 