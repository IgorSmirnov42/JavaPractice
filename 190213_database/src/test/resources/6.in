1
a
239
1
b
42
6
a
239
b
6
b
42
t
7
0
