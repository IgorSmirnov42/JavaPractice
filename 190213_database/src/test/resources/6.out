Write command name (8 for help): Write owner name: Write phone number: Write command name (8 for help): Write owner name: Write phone number: Write command name (8 for help): Write owner name: Write new phone number: Write command name (8 for help): Write owner name: Write new phone number: Write command name (8 for help): Write owner name: Write phone number: Write command name (8 for help): Write owner name: Write phone number: Write new phone number: Write command name (8 for help): a 239
b t
a 3
Write command name (8 for help): 